### PR TITLE
Add versioning

### PR DESCRIPTION
**Pull request by roznawsk**
Kindof added versioning.

It will be finished once we release `0.2.0`.
The current docs are labeled `0.2.0-beta`.

Once `0.2.0` is released we will freeze the docs for `0.2.0` in the `versioned_docs/version-0.2.0` directory.
The docs in the `/docs` directory will be the current docs, which we will actively develop. We can label them as `0.3.0-beta`.

The root of the website will be pointing to the `0.2.0` version, but there will be option to switch to `0.3.0-beta` version using the dropdown menu, those docs will reside under `/0.3.0-beta` path.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,28 +5,37 @@
...
 
 ```
 npm run build
 ```
 
 This command generates static content into the `build` directory and can be served using any static contents hosting service.
 
+## Versioning
+
+We follow the versioning system provided by docusaurus: [click](https://docusaurus.io/docs/versioning)
```
Comment (sgfn) [inline README.md:32]:
```suggestion
We follow [the versioning system provided by docusaurus](https://docusaurus.io/docs/versioning)
```

File: README.md [hunk 2]
```diff
@@ -5,28 +5,37 @@
...
+
+To create a new docs version:
+* make sure the current docs version inside `/docs` directory and `sidebars.js` are ready to be frozen
```
Comment (sgfn) [inline README.md:35]: I'd rephrase that, the way it is right now it made me look all over `/docs` and `sidebars.js` for a version string like `0.2.0` :smile:
```suggestion
* make sure that `sidebars.js` and files inside `docs/` directory are up-to-date and ready to be frozen
```

**Review by sgfn — COMMENTED** [review]
One concern: after building the docs, I got the following pop-up when selecting a version other than the one specified in `current`


This is good for past versions, but I'd imagine that if we had `current` set to `0.2.0`, and we were then working on `0.3.0-dev`, the exact same message would be shown. It's not a big issue, but if we can customise the message, I'd say that in these cases we should instead display something like "This is documentation for Jellyfish 0.3.0-dev, which is currently a work in progress and yet to be released"

Nitpicky bonus: we could delete "docs" from this phrase (right now it says "this is documentation for jellyfish documentation 1.0.0")

**Comment by roznawsk** [comment]
> One concern: after building the docs, I got the following pop-up when selecting a version other than the one specified in `current` 
> 
> This is good for past versions, but I'd imagine that if we had `current` set to `0.2.0`, and we were then working on `0.3.0-dev`, the exact same message would be shown. It's not a big issue, but if we can customise the message, I'd say that in these cases we should instead display something like "This is documentation for Jellyfish 0.3.0-dev, which is currently a work in progress and yet to be released"
> 
> Nitpicky bonus: we could delete "docs" from this phrase (right now it says "this is documentation for jellyfish documentation 1.0.0")

@sgfn  That's right, however this is due to config. Right now the `latest` version is set to current version, but once we switch the `latest` to `0.2.0`, the `0.3.0-dev` will have a message showing that this is a future release, so just what you want. Older releases will have the deprecated warning.

**Review by sgfn — APPROVED** [review]

**Review by mickel8 — APPROVED** [review]
